src/components/ReportActionItem/IOUPreview.js

```diff
@@ -33,6 +32,7 @@ const propTypes = {
     onPayButtonPressed: PropTypes.func,
 
     /** The active IOUReport, used for Onyx subscription */
+    // eslint-disable-next-line react/no-unused-prop-types
     iouReportID: PropTypes.string.isRequired,
```

Comment:
I can't see this being used now. Why can't we remove it, am I missing something?

Reply:
I also have the same thought, but `iouReportID` is used to get the report from Onyx.
https://github.com/Expensify/App/blob/a414d1a30c1726ae6dcfd44d4f28e1ccaf3b7cdf/src/components/ReportActionItem/IOUPreview.js#L227-L229

Reply:
So the IOUPrevew presumably is missing the iouReport data if we remove it, but I can't find the exact reason why. Could you please point to it so I can resolve this thread.

Reply:
I think what @mollfpr linked there is where it is used. the HOC used the id to fetch the iouReport and that is then used in the component.

Reply:
`iouReport` data is used to show the IOU data no? Like email, total, currency, and hasOutstandingIOU.

Reply:
yes

Reply:
Yeah, I was looking for prop usage and didn't spot this. Thanks